tests/wolfssl_crypto_benchmark/main.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2006-2017 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
```

Comment:
as with the files before, please adapt to our default style for copyright notes.

Reply:
reviewed all copyright notes in all the contributed files